APP/app.test.js

```diff
@@ -109,3 +109,31 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /app/articles/:article_id/comments", () => {
+  it("Gets all comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toEqual(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
```

Comment:
I would ideally want you to be more specific about what article_id you get back on each comment - you know what it should be